documentation/staging/content/userguide/istio/istio.md

```diff
@@ -142,6 +142,39 @@ If the WebLogic administration port is enabled on the Administration Server:
 Additionally, when Istio support is enabled for a domain, the operator
 ensures that the Istio sidecar is not injected into the introspector job's pods.
 
+#### Support for network changes in Istio v1.10 and later
+
+Prior to Istio release 1.10, the Istio Envoy proxy redirected all inbound traffic to the localhost
+network interface.  The network channels listed above, which the operator automatically added, 
+were bound to the localhost interface in order for WebLogic to receive the inbound traffic.  
+Starting with Istio 1.10, the networking behavior was changed in that the proxy no longer redirects 
+the traffic to the localhost interface, but instead forwards it to the network interface associated
+to the pod's IP.  In order to support Istio v1.10 and later, as well as pre-1.10 releases, the
+operator will:
+    
+* Add an additional WebLogic HTTP protocol network channel for the readiness probe that is bound to the localhost network interface.
+* Add additional WebLogic network channels, bound to the localhost network interface, for each defined custom network channel.  
+* Continue to automatically add the network channels described above in [How Istio-enabled domains differ from regular domains](#how-istio-enabled-domains-differ-from-regular-domains)
+
+When adding additional WebLogic network channels for the readiness probe and any defined custom channels,
+the name of the additional channel will be appended with '-lhNN', where NN represents
+a two digit value for uniqueness.
+
+For example, the additional WebLogic HTTP protocol network channel for the readiness probe would be
+defined as follows:
+
+|Name|Port|Listening address|Protocol|Exposed as a container port|
+|----|----|----|--------|-----|
+|`http-probe-lh01`|From configuration Istio `readinessPort` | `127.0.0.1` | `http`| No |
+
+As another example, for a custom WebLogic network channel defined as `T3Channel` with port `5556`
+and protocol `t3`, the additional channel would be defined as follows: 
+
+|Name|Port|Listening address|Protocol|Exposed as a container port|
+|----|----|----|--------|-----|
+|`T3Channel-lh01`| `5556` | `127.0.0.1` | `t3`| Yes |
+
+To learn more about changes to Istio networking beginning with Istio 1.10, see [Upcoming networking changes in Istio 1.10](https://istio.io/latest/blog/2021/upcoming-networking-changes//).
```

Comment:
Does the user need to do anything if upgrading istio from 1.9 to 1.10 or higher,  does he/she needs to re-run the introspector by updating the introspectVersion or something else?

Reply:
This is a very good question.   With this change, we can support all versions of Istio and so  I don't believe re-running introspection would make much difference or is necessary.  Do we have a documented process for upgrading Istio in a cluster?  That is, do we recommend shutting down the domain before upgrading Istio or can you upgrade Istio while a domain is running?